src/extension.ts

```diff
@@ -596,7 +596,7 @@ class RdbgAdapterDescriptorFactory implements DebugAdapterDescriptorFactory {
 					}
 				}
 				else {
-					const wait_ms = config.waitLaunchTime ? config.waitLaunchTime : 1000 /* 1 sec */;
+					const wait_ms = config.waitLaunchTime ? config.waitLaunchTime : 5000 /* 5 sec */;
```

Comment:
I'm not sure if the waiting time is not enough for all environments, but I was able to confirm it works in my environment.

Also it was enough in tests.

Reply:
https://github.com/ruby/vscode-rdbg/blob/master/src/test/suite/extension.test.ts#L302